automatic/pycharm-edu/tools/chocolateyinstall.ps1

```diff
@@ -11,7 +11,7 @@ New-Item -ItemType Directory -Force -Path $programfiles\JetBrains
  
 $packageArgs = @{
   packageName   = $env:ChocolateyPackageName
-  softwareName   = "JetBrains PyCharm Edu*"
+  softwareName   = "*PyCharm Edu*"
```

Comment:
Please remove the star in front, this is not required when the software name starts with `PyCharm Edu`

Reply:
I confirm that string "JetBrains PyCharm Edu*" doesn't match with last versions of pycharm for at least 1 year. So uninstall process doesn't work. 

Correcting SoftwareName as I did resolve the problem.

![Capture d’écran 2020-10-01 à 17 43 24](https://user-images.githubusercontent.com/24703064/94832023-b3a5b700-040d-11eb-8dac-086830330cea.png)